src/test/feedback/commands/submitFeedbackListener.test.ts

```diff
@@ -32,7 +60,7 @@ describe('submitFeedbackListener', function () {
 
         when(mockTelemetry.postFeedback(anything())).thenThrow(new Error(error))
 
-        const webview = new FeedbackWebview(instance(mockTelemetry))
+        const webview = new FeedbackWebview(instance(mockTelemetry), 'CodeWhisperer')
```

Comment:
The entire test code here is duplicated except for a single value. If this really needs to be tested, the test should be parameterized or deduplicated somehow.

Reply:
Reused the code into one test case.